tuxemon/states/combat/combat_animations.py

```diff
@@ -585,13 +585,22 @@ def animate_parties_in(self) -> None:
             left=w,
         )
 
-        combat_back = player.template[0].sprite_name
-        player_back = self.load_sprite(
-            f"gfx/sprites/player/{combat_back}_back.png",
-            bottom=front_island.rect.centery + scale(6),
-            centerx=front_island.rect.centerx,
-        )
-        assert player_back
+        combat_back = player.template[0].combat_front
+        filename = f"gfx/sprites/player/{combat_back}_back.png"
+        try:
+            player_back = self.load_sprite(
+                filename,
+                bottom=front_island.rect.centery + scale(6),
+                centerx=front_island.rect.centerx,
+            )
+        except:
+            logger.warning(f"This {filename} doesn't exist.")
```

Comment:
The sentence doesn't really make sense.
Let say my file is named "HelloWorld.txt".
Then the message logged would be: `This HelloWorld.txt doesn't exist.` 
It sounds weird, I would replace "This" by "File" or the whole sentence by "(File) {filename} cannot be found", something like this.

Reply:
whole sentence done